src/index.ts

```diff
@@ -4,20 +4,32 @@ import { exec } from "child_process";
 import { join } from "path";
 
 export default function run(commandArgs: string[]) {
```

Comment:
For unit tests, we should add 2 tests that assert the default branch use for `exec` first arg and the provided argument when targeting another branch.

Same for the Stryker's configuration path.

Are you familiar with unit testing and mocks ? Or would you like an example with one of the test case involved ?